tests/scripts/ci_helpers.sh

```diff
@@ -25,6 +25,45 @@ function install_deps() {
     date
 }
 
+function install_juju_simple() {
+    sudo snap install juju
+    mkdir -p ~/.local/share/juju
+    juju bootstrap localhost
+}
+
+function setup_juju_spaces() {
+    set -ex
+    date
+    juju add-model spacetest
+    juju add-space cluster
+    # Subnet value from LXD profile.
+    juju move-to-space cluster 10.85.4.0/24
+}
+
+function verify_juju_spaces_config() {
+    set -ex
+    date
+    local public="10.196.231.0/24"
+    local cluster="10.85.4.0/24"
+
+    # Verify the cluster network is correct subnet.
+    output=$(juju ssh microceph/0 -- "sudo microceph cluster config get cluster_network")
+    echo $output | grep $cluster
```

Comment:
Wonder if we could tighten the check here. For instance, check that `$cluster` is returned exactly once?

Reply:
Since this is a `cluster get` request, it will only return a single subnet (in a table formatted response). We can check for single instance of the subnet in the return string but we can't really have more than one value there so it would be redundant.